pkgs/development/python-modules/pysnow/default.nix

```diff
@@ -15,10 +16,31 @@ buildPythonPackage rec {
   pname = "pysnow";
   version = "0.7.16";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "5df61091470e48b5b3a6ea75637f69d3aacae20041487ea457a9a0e3093fba8c";
-  };
+  srcs = [
+    # for the actual source. the github source is in painful poetry format.
+    (fetchPypi {
+        inherit pname version;
+        sha256 = "5df61091470e48b5b3a6ea75637f69d3aacae20041487ea457a9a0e3093fba8c";
+    })
+    # for the tests - important because we make some dependency version subsitutions
```

Comment:
you should be able to just use:
```
format = "pyproject";
```

and it should be able to build from source fine

Reply:
ah so it _does_ work that way, but it has the side effect of disabling it for py27.

Reply:
I think that's fine, python2 has been EOL for 9 months now